src/bundle/Resources/public/scss/_alloyeditor-anchor-edit.scss

```diff
@@ -0,0 +1,50 @@
+.ez-ae-anchor-edit {
+    display: flex;
+    padding: calculateRem(8px) 0 calculateRem(8px) calculateRem(8px);
+
+    &__input-wrapper {
+        margin-right: calculateRem(10px);
+    }
+
+    &__input-label {
+        font-weight: 700;
+    }
+
+    &__input {
+        width: calculateRem(400px);
+    }
+
+    &__actions {
+        display: flex;
+        align-items: flex-end;
+        justify-content: space-around;
+        width: calculateRem(110px);
+    }
+
+    &__btn {
+        &--trash,
+        &--save {
+            border-radius: 50%;
+            width: calculateRem(40px);
+            height: calculateRem(40px);
+            padding: 0;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            cursor: pointer;
+        }
+
+        &--trash {
+            background-color: $ez-color-base-dark;
+        }
+
+        &--save {
+            background-color: $ez-color-positive;
+        }
+
+        &[disabled] {
```

Comment:
I'm still thinking about unifying this. Because I found we're adding the similar rule here and there in our codebase. We have to check whether we could unify it and make it more generic.